.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
```

Comment:
I haven't seen a .editorconfig before (but I haven't seen a lot of things).  Could you explain a little what it's doing and its purpose? Thank you!

Reply:
It's a standard that defines basic text editing conventions (tabs vs spaces, line endings, etc) for a project.  They have plugins for more or less every text editor I've ever heard of that implement the preferences defined.  I tried to capture the conventions already used by the project, which will make sure future contributions (from people with an EditorConfig plugin, at least) will match.

Reply:
awesome thanks